Prox/Prox/Utilities/TravelTimesProvider.swift

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import CoreLocation
+
+var travelTimePadding: Double = {
+    return RemoteConfigKeys.travelTimePaddingMins.value * 60
+}()
+
+var MIN_WALKING_TIME: Int = {
+    // Note that this is semantically maximum walking time,
+    // rather than minimum walking time (as used throughout the codebase).
+    return RemoteConfigKeys.maxWalkingTimeInMins.value
+}()
+
+var YOU_ARE_HERE_WALKING_TIME: Int = {
```

Comment:
again, I think this is legacy copied over from the original implementation but I shall address